ethereum/src.ts/deploy.ts

```diff
@@ -104,9 +104,9 @@ export class Deployer {
                 this.addresses.ZkSync.GovernanceFacet
             )
         );
-        const genesisBlockHash = getHashFromEnv('CONTRACTS_GENESIS_ROOT'); // TODO: confusing name
+        const genesisBatchHash = getHashFromEnv('CONTRACTS_GENESIS_ROOT'); // TODO: confusing name
         const genesisRollupLeafIndex = getNumberFromEnv('CONTRACTS_GENESIS_ROLLUP_LEAF_INDEX');
-        const genesisBlockCommitment = getHashFromEnv('CONTRACTS_GENESIS_BLOCK_COMMITMENT');
+        const genesisBatchCommitment = getHashFromEnv('CONTRACTS_GENESIS_BATCH_COMMITMENT');
```

Comment:
Do we really want to change this? Then we would need to change server specifically for this small change

Reply:
For consistency yes. For production environments it won't matter, since genesis-related stuff is needed only for regenesis